### PR TITLE
Minor cleanup on multiple definitions of the same function

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
Just spotted I could remove 500 LoC just by using an import, figured it was a good deal.

### CODE REVIEW

File: src/ocean/text/utf/UtfUtil.d [hunk 1]
```diff
@@ -34,13 +34,10 @@
 
 module ocean.text.utf.UtfUtil;
 
-
```
Comment (gavin-norman-sociomantic) [inline src/ocean/text/utf/UtfUtil.d:37]: It'd really help review if you didn't meld whitespace changes in with other types of change.

**Review by gavin-norman-sociomantic — APPROVED** [review]
Ha, well spotted!

**Review by mihails-strasuns-sociomantic — COMMENTED** [review]
It looks like there are not test cases for unicode errors in these modules  - would it be simple enough to copy some from phobos?

**Comment by mathias-lang-sociomantic** [comment]
The only test I could find was: https://github.com/dlang/druntime/blob/3485ff859a29ba44e7949bc49e62d5dd3a2a9ff0/src/rt/lifetime.d#L2070-L2096
Is it really worth it ?

**Comment by mihails-strasuns-sociomantic** [comment]
I was thinking about something like https://github.com/dlang/phobos/blob/feab9a2f5225d7f3a5179f24ae3699e64455807c/std/utf.d#L1532-L1555 but if you feel it is too much trouble, nevermind.

**Comment by mathias-lang-sociomantic** [comment]
Given this code is never ever touched, I'd say let's skip it. I'd rather spend time of getting rid of this code than testing it :)

**Comment by mihails-strasuns-sociomantic** [comment]
Reasonable enough :)

**Pull request merged**